baybe/searchspace/discrete.py

```diff
@@ -298,52 +308,113 @@ def from_simplex(
 
         # Validate non-negativity
         min_values = [min(p.values) for p in simplex_parameters]
+        max_values = [max(p.values) for p in simplex_parameters]
         if not (min(min_values) >= 0.0):
             raise ValueError(
                 f"All parameters passed to '{cls.from_simplex.__name__}' "
                 f"must have non-negative values only."
             )
 
-        def drop_invalid(df: pd.DataFrame, max_sum: float, boundary_only: bool) -> None:
-            """Drop rows that violate a specified simplex constraint.
+        def drop_invalid(
+            df: pd.DataFrame,
+            max_sum: float,
+            boundary_only: bool,
+            min_nonzero: Optional[int] = None,
+            max_nonzero: Optional[int] = None,
+        ) -> None:
+            """Drop rows that violate the specified simplex constraint.
 
             Args:
                 df: The dataframe whose rows should satisfy the simplex constraint.
                 max_sum: The maximum row sum defining the simplex size.
                 boundary_only: Flag to control if the points represented by the rows
                     may lie inside the simplex or on its boundary only.
+                min_nonzero: Minimum number of nonzero parameters allowed per row.
```

Comment:
Isn't this rather `Minimum number of nonzero parameters required per row`? Or do I understand this wrong?

Reply:
i think its good to say the minimum is required and theres a maximum allowed so i changed it